scripts/acn/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:18.04
+
+USER root
+
+RUN apt-get update && apt-get upgrade -y
+
+RUN apt install -y python3 python3-pip
+
+# utils
+RUN apt install -y wget subversion
+
+# golang
+RUN wget https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz && \
+  tar -xzvf go1.13.8.linux-amd64.tar.gz -C /usr/local && \
+  export PATH=$PATH:/usr/local/go/bin && echo 'export PATH=$PATH:/usr/local/go/bin' >> ~/.bashrc && \
+  mkdir $HOME/go
+
+ENV PATH="${PATH}:/usr/local/go/bin"
+
+# deployment scripts deps: needed only if configuration checks are enabled
+RUN python3 -m pip install pymultihash ecdsa base58
+
+WORKDIR /acn/
+
+# get node source code
+RUN svn export https://github.com/fetchai/agents-aea/branches/feature/add-standalone-acn-node/packages/fetchai/connections/p2p_libp2p/ /acn/node
+
+# get deployment script
+RUN svn export https://github.com/fetchai/agents-aea/branches/feature/add-standalone-acn-node/scripts/acn/run_acn_node_standalone.py /acn/
```

Comment:
This should point to master branch; when we release later this will otherwise be broken

Reply:
yes, changing link to master is part of the TODO list (see description of pr)